### PR TITLE
Fix tests for SF >= 3.3.10

### PR DESCRIPTION
**Pull request by jlamur**
Closes https://github.com/sonata-project/SonataAdminBundle/issues/4685

## Subject

The build is broken with SF >= 3.3.10.
Caused by https://github.com/symfony/symfony/pull/24431/files?diff=unified#diff-064e58dbe11951327d614e6922caaae5R234

### CODE REVIEW

**Comment by greg0ire** [comment]
Thanks @jlamur !

File: Tests/Controller/CRUDControllerTest.php [hunk 1]
```diff
@@ -143,26 +143,33 @@ protected function setUp()
...
             ->getMock();
 
+        $templatingRenderReturnCallback = $this->returnCallback(function (
+            $view,
+            array $parameters = array(),
+            Response $response = null
+        ) use (
+            &$params,
+            &$template
+        ) {
```
[the remark this answers is not in the document]
Comment (ElectricMaxxx) [reply Tests/Controller/CRUDControllerTest.php]: And it is a test.

[the remark this answers is not in the document]
Comment (greg0ire) [reply Tests/Controller/CRUDControllerTest.php]: What does this have to do with the 80/120 char limit? And why should test not follow the rules?

```php
        $templatingRenderReturnCallback = $this->returnCallback(function (
+            $view,
+            array $parameters = array(),
+            Response $response = null
+        ) use (&$params, &$template) {
```
is still largely under 80

[the remark this answers is not in the document]
Comment (jlamur) [reply Tests/Controller/CRUDControllerTest.php]: BTW, just to mention it, I did not write this code :kissing: :notes:

**Pull request merged**